### PR TITLE
Match API changes for contact

### PR DESCRIPTION
**Pull request by benweatherman**
Updated URL and data to accomodate the structure returned by the API

### CODE REVIEW

**Comment by benweatherman** [comment]
🚨This should probably just be closed since #55 is probably the way we'll go forward 🚨

File: src/pages/Service.js [hunk 1]
```diff
@@ -32,13 +32,23 @@ class Service extends Component {
...
     axios
-      .get(`${process.env.REACT_APP_CMS_ENDPOINT}/pages/${this.props.match.params.id}?fields=content,extra_content,topic(text),locations(location(name,street,city,state,zip,country,hours)),contacts(contact(name,email,phone))`)
+      .get(`${process.env.REACT_APP_CMS_ENDPOINT}/pages/${this.props.match.params.id}?fields=content,extra_content,topic(text),contacts(contact(name,email,phone,hours,location(name,street,city,state,zip,country)))`)
       .then(res => {
         this.setState({ data: res.data })
       })
       .catch(err => console.log(err))
   }
 
+  cleanContacts(contacts) {
```
Comment (ifsimicoded) [inline src/pages/Service.js]: no need to add an additional method. We can just update address to location in the JSX and update Contact.js to reference contact.contact. I can make this update in another branch. Or also graphql!

Comment (benweatherman) [reply src/pages/Service.js]: I was trying to keep the sub-components from changing, but up to y'all how to structure stuff. Do you want me to make the change in the JSX for the graphql?

**Comment by benweatherman** [comment]
I'm gonna close in lieu of the graphql version

**Pull request closed without merging**